frame/example-offchain-worker/README.md

```diff
@@ -7,7 +7,7 @@ concepts, APIs and structures common to most offchain workers.
 Run `cargo doc --package pallet-example-offchain-worker --open` to view this module's
 documentation.
 
-- [`pallet_example_offchain_worker::Config`](./trait.Config.html)
+- [`pallet_example_offchain_worker::Config`](./trait.Trait.html)
```

Comment:
```suggestion
- [`pallet_example_offchain_worker::Trait`](./trait.Trait.html)
```
nothing important in this suggestion but it is more consistent with other README